frontend/src/components/CovidTestCard.js

```diff
@@ -12,45 +39,83 @@ export default function CovidTestCard({covidTest}) {
 		PCR_TEST: 'PCR',
 	};
 
+	const handleClickOpen = () => {
+		setOpen(true);
+		setNewCovidTest(covidTest);
+	};
+
+	const handleClickClose = () => {
+		setOpen(false);
+	};
+
+	const handleSave = () => {
+		editCovidTest(covidTestToEdit, token).catch(console.error);
+		setNewCovidTest(initialCovidTest);
+		getCovidTests(token)
+			.then((covidTests) => setAllCovidTests(covidTests))
+			.catch(console.error);
+		handleClickClose();
+	};
+
 	return (
-		<StyledCard>
-			<CardContent style={{padding: '10px', fontSize: 'small'}}>
-				<Typography variant='overline' style={{color: '#303030', fontSize: 10}}>
-					{testTypes[covidTest.testType]}
-				</Typography>
-				<Typography
-					variant='subtitle2'
-					style={{color: '#233d4dff', marginBottom: 8, fontSize: 10}}
-				>
-					{moment(covidTest.dateTime).format('llll')}
-				</Typography>
-				<Typography
-					variant='body2'
-					style={{
-						fontWeight: 500,
-						fontSize: 10,
-						lineHeight: 1,
-						color: '#303030',
-						marginBottom: 4,
-					}}
-				>
-					{covidTest.result}
-				</Typography>
-			</CardContent>
+		<>
+			<StyledCard>
+				<CardContent style={{padding: '10px', fontSize: 'small'}}>
+					<Typography
+						variant='overline'
+						style={{color: '#303030', fontSize: 10}}
+					>
+						{testTypes[covidTest.testType]}
+					</Typography>
+					<Typography
+						variant='subtitle2'
+						style={{color: '#233d4dff', marginBottom: 8, fontSize: 10}}
+					>
+						{moment(covidTest.dateTime).format('llll')}
```

Comment:
consider this
https://momentjs.com/docs/#/-project-status/

Reply:
Interesting!